### PR TITLE
Fix the versions of all tools installed in the build container

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -24,21 +24,16 @@ jobs:
...
-        name: assisted-installer
+        name: assisted-installer-build
         username: ${{ secrets.OCPMETAL_USERNAME }}
         password: ${{ secrets.OCPMETAL_PASSWORD }}
         REGISTRY: 'quay.io/ocpmetal'
-        dockerfile: Dockerfile.assisted-installer
-        buildargs: GIT_REVISION
-        tags: "stable,${{ env.GIT_REVISION }}"
+        dockerfile: Dockerfile.assisted-installer-build
+        tags: "ronnie"
```
Comment (eranco74) [inline .github/workflows/build.yml:36]: Ronnie?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -24,21 +24,16 @@ jobs:
...
         echo "{}" > ~/.docker/config.json
         touch ${HOME}/.gitconfig
-    - name: build
-      run: |
-        skipper make
-    - name: Get release version
-      id: get_version
-      run: echo ::set-env name=GIT_REVISION::$(echo ${GITHUB_SHA})
-    - name: Publish to Registry
+    - name: Publish build to Registry
```
Comment (eranco74) [inline .github/workflows/build.yml:27]: I don't understand this change.
What are you trying to do here?

File: Dockerfile.assisted-installer-build [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+FROM golang:1.14.3
+ENV GO111MODULE=on
+
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
-RUN go get -u golang.org/x/tools/cmd/goimports
-RUN go get -u github.com/onsi/ginkgo/ginkgo  # installs the ginkgo CLI
-RUN go get -u github.com/onsi/gomega/...     # fetches the matcher library
-RUN go get -u github.com/golang/mock/mockgen
-RUN apt-get update && apt-get install -y docker.io \
+RUN go get -u golang.org/x/tools/cmd/goimports@v0.0.0-20200520220537-cf2d1e09c845
```
Comment (eranco74) [inline Dockerfile.assisted-installer-build:5]: :+1:

File: .github/workflows/build.yml (hunk 2 above)
Comment (ronniel1) [reply .github/workflows/build.yml]: This was just for a test for the issues that we had on Wednesday

**Pull request merged**